tools/Streetview/sv-wizard/Gruntfile.js

```diff
@@ -0,0 +1,464 @@
+// Generated on 2015-12-06 using
```

Comment:
In general we shouldn't put generated file in the repo, it's not really part of the code itself. You could create a setup.sh script maybe that will run all required generators.